packages/fxa-admin-server/src/scripts/audit-tokens.ts

```diff
@@ -58,9 +58,10 @@ const tables = {
 //#endregion
 
 //#region Result Handling
-function formatStatLabel(label: string) {
+export function formatStatLabel(label: string) {
   let cleaned = label;
   cleaned = cleaned.replace(/(fxa|fxa_profile|fxa_oauth)\./g, '$1_');
+  cleaned = cleaned.replace(/\.+/g, '.');
```

Comment:
Random regex Q: Is there a difference/benefit between `\.+` versus `\.{2,}`?
- https://regexr.com/7c3vu uses `\.+` and seems to have 9 matches (although 7 seem to replace `.` with `.`).
- https://regexr.com/7c3vr uses `\.{2,}` and seems to only have 2 matches, `...` and `..`.

Reply:
.+ is one or more.  .{2,} is two or more